File: FantasyFootball_Backend/Player.cs
namespace FantasyFootball;

using System;

public class Player
{
    public string? name { get; }
    public string? team { get; }
    public string? position { get; }

    public Player(string name, string team, string position)
    {
        this.name = name;
        this.team = team;
        this.position = position;
    }
}

File: backend/Player.py
class Player:
    def __init__(self, name, position, team):
        self.name = name
        self.position = position
        self.team = team

    def __eq__(self, other):
        if isinstance(other, Player):
            return (self.name, self.position, self.team) == (other.name, other.position, other.team)
        return False

    def __hash__(self):
        return hash((self.name, self.position, self.team))

    def __repr__(self):
        return f"Player(name={self.name}, position={self.position}, team={self.team})"
